src/pages/EntitiesSearchPage/EntitiesSearchPage.jsx

```diff
@@ -81,6 +115,24 @@ function EntitiesSearch() {
             api.search();
         }
     }, [searchParams]);
+    useEffect(() => {
+        // handle sorting separately in order to only update in case of changes
+        if (sortState) {
+            const dir = sortState.direction === 1 ? "asc" : "desc";
+            const sortBy = `${sortState.field}_${dir}`;
```

Comment:
```suggestion
            let sortBy = sortState.field;
            if (sortState.direction) {
                const dir = sortState.direction === 1 ? "asc" : "desc";
                sortBy = `${sortState.field}_${dir}`;
            }
```

Reply:
Implemented this change, both here and in the Letters search 👍

Reply:
> Shower thought on this one: force the sort to relevance only when you submit the keyword search form or click the submit button for it. If that works, you can leave the default at alphabetic label and discard the two related commit suggestions I made. You'll still need the suggested change for the `sortState.direction` fix.

@blms Good idea – I've added a change that I think accomplishes this. What do you think?